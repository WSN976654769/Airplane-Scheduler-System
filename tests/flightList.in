flight import flights.csv
location import locations.csv
locations
flights import flights.csv
flight import flights.csv
flights
flight 0
flight 1
exit