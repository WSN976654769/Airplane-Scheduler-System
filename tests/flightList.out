User: Imported 0 flights.
3 lines were invalid.

User: Imported 51 locations.

User: Locations (51):
AbuDhabi, Adelaide, AliceSprings, Alta, Athens, Auckland, Beijing, Berlin, Bern, Bordeaux, Brisbane, Cairo, Cardiff, Chicago, CoffsHarbour, Dallas, Darwin, Dubai, Dubbo, GoldCoast, Hanoi, Hobart, Houston, Jakarta, Johannesburg, Lagos, Liverpool, London, Longyearbyen, LosAngeles, Luton, Madrid, Manchester, Moscow, NewYork, Orange, Oslo, Paris, Perth, Rio, Rome, SanFrancisco, Stockholm, Sydney, Toulouse, Townsville, Tromso, Ufa, Utqiagvik, Vladivostok, Washington

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
(None)

User: Imported 2 flights.
1 line was invalid.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
   0 Mon 19:00   Mon 20:27   Sydney --> Hobart
   1 Mon 21:30   Mon 22:57   Sydney --> Hobart

User: Flight 0
Departure:    Mon 19:00 Sydney
Arrival:      Mon 20:27 Hobart
Distance:     1,045km
Duration:     1h 27m
Ticket Cost:  $272.00
Passengers:   29/120

User: Flight 1
Departure:    Mon 21:30 Sydney
Arrival:      Mon 22:57 Hobart
Distance:     1,045km
Duration:     1h 27m
Ticket Cost:  $272.00
Passengers:   29/120

User: Application closed.
